Example/ViewController.swift

```diff
@@ -215,3 +223,30 @@ extension ViewController: OktaNetworkRequestCustomizationDelegate {
         }
     }
 }
+
+extension UIViewController {
+    static func displayErrorMessage(_ error: OktaOidcError) -> String {
```

Comment:
Having a static function on UIViewController to return an error string doesn't feel like the right place for this. 

Perhaps this should be an extension to `OktaOidcError`?  Such as

```
extension OktaOidcError {
    var displayMessage: String {
        switch self { ... }
    }
}
```

Then, when using this up above in the `showError` method, you can simply use `tokenView.text = oidcError.displayMessage`.

Reply:
Done.